cli/src/cli/init.rs

```diff
@@ -75,7 +102,7 @@ pub fn init(path: &PathBuf) -> DefaultResult<()> {
     let ignores = [
         &DIST_DIR_NAME,
         &TEST_DIR_NAME,
-        &DEFAULT_BUNDLE_FILE_NAME,
+        format!("*.{}", DNA_EXTENSION).as_str(),
```

Comment:
you'll only need this here, if it's still going to be building the file in the root of the app, instead of in the DIST folder

Reply:
fixed.